src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,16 +100,22 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        int compare = LocalDate.now().compareTo(someDate);
+        return compare == 0 ? someDate.toString() + " is today"
+                : compare < 0 ? someDate.toString() + " is after " + LocalDate.now().toString()
+                : someDate.toString() + " is before " + LocalDate.now().toString();
```

Comment:
let's use isAfter() or isBefore()  those are more relevant

Reply:
got it